tests/test_ssl.py

```diff
@@ -46,8 +46,11 @@ def on_close(self):
 
 def test_success():
     n = network.Server()
+    cert_dir = os.path.dirname(__file__) + '/cert/'
+    certfile = cert_dir + 'cert.pem'
+    keyfile = cert_dir + 'key.pem'
     n.add_server(PORT, network.BasicHandler, ssl=True,
-                 ssl_certfile='cert/cert.pem', ssl_keyfile='cert/key.pem')  # self-signed certs
+                 ssl_certfile=certfile, ssl_keyfile=keyfile)  # self-signed certs
```

Comment:
this change breaks this test if the certfile and keyfile are the ones in the cert directory, no?